packages/fx-core/src/component/generator/openApiSpec/customEngineAgentGenerator.ts

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+/**
+ * @author yuqzho@microsoft.com, Ning Tang
```

Comment:
I would suggest setting the file owner correctly. thought this might be @KennethBWSong ?

Reply:
Updated.